ckanext/opendatani/controller.py

```diff
@@ -269,17 +268,20 @@ def prepare_report(self, org):
         try:
             data_dict = {'org_name': org}
 
-            if org == 'full-portal':
+            if org == '@complete':
                 data_dict = {}
+                org = org[1:]
+            else:
+                org = 'org-' + org
```

Comment:
This just prepends 'org' to the org name for clarity

Reply:
I'm not quite understanding this. Can you include this as an actually comment in the code? (So that future you or me is not confused :) )